### ISSUE TITLE
Support for confirmation emails in different languages

### ISSUE THREAD

**Issue by gusreyes01** [timekit-io/booking-js#91]
I think there should be a setting so confirmation emails are sent in different languages (In my case Spanish) however a setting so they can be overriden and/or disabled in order to use my own email backend could work as well.

**Comment by laander** [comment]
@gusreyes01 Good idea. Multi-language support in the widget is pretty far, but the emails could use some settings. Added to feature request roadmap 👍

**Comment by gusreyes01** [comment]
Thanks, looking forward to it!
